src/css/layout.css

```diff
@@ -1,3 +1,14 @@
+html {
+	height: 100%;
+	min-height: 100%;
+}
+
+body {
+	min-height: 100%;
+	display: grid;
+	grid-template-rows: auto 1fr auto;
+}
+
```

Comment:
I'm wondering if it would be helpful for me to write a style guide (after everything else critical is written, perhaps) about the conventions and organization strategies for cap static. The `layout.css` file is specifically intended for layout classes — that are prepended with `l-`. 

For example, `l-interiorPage`

Reply:
👍 moved!